src/components/checkbox/checkbox.ts

```diff
@@ -76,6 +88,10 @@ export class MCheckbox extends ModulVue {
         return this.position === MCheckboxPosition.Left;
     }
 
+    private get hasCheckboxAlign(): MCheckboxVerticalAlignement {
```

Comment:
Le nom ici est pas bon.  Ça devrait être simplement verticalAlign
Je te suggère aussi de te faire des get "helper" pour tes conditions dans le html
private get isAlignTop return this.verticalAlign === MCheckboxVerticalAlignement.Top

Reply:
Je l'ai changé, tu pourras valider que c'est correct.